### PR TITLE
feat(rns): new placement events

### PR DESCRIPTION
**Pull request by leandroyalet**

### CODE REVIEW

File: src/rns/hooks/utils.ts [hunk 1]
```diff
@@ -1,5 +1,17 @@
 import { HookContext } from '@feathersjs/feathers'
 
+export function setSellerAddressParamHook (context: HookContext): HookContext {
```
Comment (AuHau) [inline src/rns/hooks/utils.ts:3]: Sorry about this, my bad. Not sure if we have to have the `utils.ts` file anymore as it was mainly in place to de-duplicate code, but we can keep it for readability I guess.

File: src/rns/rns.blockchain.ts [hunk 2]
```diff
@@ -73,67 +75,85 @@ async function nameChangedHandler (eventData: EventData): Promise<void> {
...
-      throw new Error(`Domain with token ID ${tokenId} not found!`)
-    }
+  const transactionHash = eventData.transactionHash
+  const tokenId = Utils.numberToHex(eventData.returnValues.tokenId)
+
+  const lastOffer = await DomainOffer.findOne({ where: { tokenId: tokenId, status: 'ACTIVE' } })
+
+  if (lastOffer) {
+    logger.info(`Found last offer for ${tokenId}`)
+    lastOffer.status = 'CANCELED'
```
Comment (AuHau) [inline src/rns/rns.blockchain.ts:133]: Shouldn't there be different status than `CANCELED` for sold domains?

Comment (leandroyalet) [reply src/rns/rns.blockchain.ts:133]: have a good eye, I tried to reflect the Placement contract behavior and to avoid duplicate the sold state

**Comment by AuHau** [comment]
Sweet, thanks Leandro!

**Pull request merged**